bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -213,7 +212,7 @@ func getSA(name string, nameSuffix string, project string) string {
 func (gcp *Gcp) GetK8sConfig() (*rest.Config, *clientcmdapi.Config) {
 	// TODO(jlewi): Should we unify the code by just setting ts and then calling
 	// ts.Tpken to get a token?
-	accessToken, _ := gcp.kfDef.GetSecret(GcpAccessTokenName)
+	accessToken, _ := gcp.kfDef.GetEncodedSecret(GcpAccessTokenName)
```

Comment:
Revert. We need the raw accessToken not the bcrypted value token. If we use the bcrypted value token it won't work.